package.json

```diff
@@ -71,6 +77,7 @@
     "clean": "rm -rf packages/wonder-stuff-*/dist && rm -rf packages/wonder-stuff-*/node_modules",
     "coverage": "yarn run jest --coverage",
     "flow:ci": "flow check --max-workers 0",
+    "format": "prettier --write .",
```

Comment:
**question:** Why was this needed? Does this mean eslint will no longer apply prettier formatting in its autofixes?

Reply:
I forgot that I could've used `eslint --fix` to the formatting. 🙃